usr/share/rear/rescue/default/850_save_sysfs_uefi_vars.sh

```diff
@@ -61,7 +61,12 @@ if [[ ! -f ${UEFI_BOOTLOADER} ]]; then
     UEFI_BOOTLOADER=$(find /boot/efi -name "elilo.efi" | tail -1)
 fi
 
-# triple check it
+# in case we have a 64-bit systemd bootloader then we might be lucky with next statements
+if [[ ! -f ${UEFI_BOOTLOADER} ]]; then
+    UEFI_BOOTLOADER=$(find /boot/EFI -name "systemd-bootx64.efi" | tail -1)
```

Comment:
Interesting, looks like shifted one directory upwards ...

Just because of my curiosity, can you tell me:
1. where is your vfat (EFI) partition mounted (`mount | grep fat`)
2. do `ls -alR /boot/EFI

Thanks 

V.